.gitignore

```diff
@@ -40,3 +40,5 @@ autom4te.cache
 /play
 *.orig
 /compile_commands.json
+/src/cxx_supportlib/vendor-modified/test.cpp
```

Comment:
It appears this is a test that should not be committed.

Reply:
it's a file generated during the copy_boost_headers script, while it is cleaned up at the end, I wanted to ignore it so that as I watch the changes in my git gui, there isn't a test file popping in and out of existence.